aten/src/ATen/ExpandUtils.h

```diff
@@ -174,4 +174,19 @@ static inline bool is_expandable_to(IntArrayRef shape, IntArrayRef desired) {
   return true;
 }
 
+// True if the input two shapes are broadcastable from one to the other in any direction
+static inline bool are_expandable(IntArrayRef shape1, IntArrayRef shape2) {
```

Comment:
infer_size already does this; if the error message is a problem, refactor that code instead of introducing code that does the same logic.